packages/components/menu/src/menu-radio-item.ts

```diff
@@ -0,0 +1,58 @@
+import { computed, defineComponent, h, mergeProps, reactive, toRefs } from 'vue'
+import { primitiveProps } from '@oku-ui/primitive'
+import { composeEventHandlers } from '@oku-ui/utils'
+import { reactiveOmit, useForwardRef } from '@oku-ui/use-composable'
+import { getCheckedState } from './utils'
+import { OkuMenuItem } from './menu-item'
+import type { MenuRadioItemEmits, MenuRadioItemNativeElement } from './props'
+import { MENU_RADIO_ITEM_NAME, itemIndicatorProvider, menuRadioItemProps, scopedMenuProps, useRadioGroupInject } from './props'
+
+const menuRadioItem = defineComponent({
+  name: MENU_RADIO_ITEM_NAME,
+  components: {
+    OkuMenuItem,
+  },
+  inheritAttrs: false,
+  props: {
+    ...menuRadioItemProps.props,
+    // ...primitiveProps,
```

Comment:
When do we use it, or do we use it in all components?

Reply:
We only add it when we see the radix on the right.

![image](https://github.com/oku-ui/primitives/assets/38668796/b7df15dd-6120-47ed-87ab-4053021e347d)